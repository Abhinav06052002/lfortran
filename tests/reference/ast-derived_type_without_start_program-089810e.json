{
    "basename": "ast-derived_type_without_start_program-089810e",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/parser/derived_type_without_start_program.f90",
    "infile_hash": "65929881cfd40bdfe37506397929a4a00b6c8ed662c78f1156ea3d32",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-derived_type_without_start_program-089810e.stdout",
    "stdout_hash": "6fc12ff040a43e9826f981dd08dd6da9fbaee18d75b3d40fe00bbe3a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}