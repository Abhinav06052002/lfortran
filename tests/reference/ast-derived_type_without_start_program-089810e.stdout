(TranslationUnit
    [(Program
        __xx_main
        ()
        []
        [(ImplicitNone
            []
            ()
        )]
        [(DerivedType
            test_t
            []
            ()
            []
            [(Declaration
                (AttrType
                    TypeReal
                    []
                    ()
                    ()
                    None
                )
                []
                [(a
                []
                []
                ()
                ()
                None
                ())]
                ()
            )]
            []
        )]
        []
        []
    )]
)
